tests/compose/fetchmail/mailu.env

```diff
@@ -83,6 +83,9 @@ RELAYNETS=
 # Will relay all outgoing mails if configured
 RELAYHOST=
 
+# Show fetchmail functionality in admin interface
+FETCHMAIL_ENABLED=false
```

Comment:
Should fetchmail not be enabled in mailu.env (tests/compose/fetchmail/mailu.env)  for the fetchmail test?
Maybe the fetchmail_delay could also be lowered to 15 seconds. 
I know the fetchmail test only starts all containers, but this at least tests if the fetchmail container can start fine and connect to admin to retrieve the fetchmail pull settings.

Reply:
Hmm. Don't know - where is the test code?
I'll change it. Maybe we should update all test cases to the new files created by the setup utility before the next release.